db/migrate/20151118185013_backfill_gift_card_line_items.rb

```diff
@@ -0,0 +1,16 @@
+class BackfillGiftCardLineItems < ActiveRecord::Migration
+  def up
+    gc_variant_ids = Spree::Product.where(gift_card: true).flat_map(&:variants).flat_map(&:id)
```

Comment:
i think this can be `Spree::Variant.joins(:spree_products).where(product: { gift_card: true })`